wg-deployment/charter.md

```diff
@@ -0,0 +1,51 @@
+# WG Deployment Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+Manage manifests for deploying and operating Kubeflow.
+Develop and maintain tools like kfctl (CLI + operator) for using the manifests to deploy Kubeflow.
+
+### In scope
+
+#### Responsibilities
+
```

Comment:
Can you add some additional clarification here? I think this reads like all the responsibilities listed in the subsections below are being assumed by this WG.  When in fact its just the opposite. Maybe move this into the cross cutting section and expand on how you intend to work with the individual WG charters?

Reply:
Yeah, that was my intention. Let me change that